kubectl-cloudflow/domain/cloudflowapplication.go

```diff
@@ -124,6 +129,12 @@ type CloudflowApplicationSpec struct {
 	LibraryVersion string            `json:"library_version,omitempty"`
 }
 
+// Version of the application descriptor format
+const Version = "1"
+
+// LibraryVersion specifies cloudflow release version
+const LibraryVersion = "1.3.1-SNAPSHOT"
```

Comment:
How will we update this?

Reply:
I was thinking of pulling all such mutable constants (huh! oxymoron) in a single file

Reply:
hehe